projects/storefrontlib/src/lib/cart/add-to-cart/added-to-cart-dialog/added-to-cart-dialog.component.spec.ts

```diff
@@ -1,34 +1,50 @@
 import { async, ComponentFixture, TestBed } from '@angular/core/testing';
-import { FormsModule } from '@angular/forms';
+import { FormsModule, ReactiveFormsModule } from '@angular/forms';
 import { RouterTestingModule } from '@angular/router/testing';
 import { By } from '@angular/platform-browser';
 import { CartService } from '@spartacus/core';
 
-import { NgbModule, NgbActiveModal } from '@ng-bootstrap/ng-bootstrap';
+import { NgbActiveModal, NgbModule } from '@ng-bootstrap/ng-bootstrap';
 
-import { of, BehaviorSubject } from 'rxjs';
+import { BehaviorSubject, of } from 'rxjs';
 
 import { SpinnerModule } from './../../../ui/components/spinner/spinner.module';
 
 import { AddedToCartDialogComponent } from './added-to-cart-dialog.component';
 import {
+  Component,
   DebugElement,
+  Input,
+  NO_ERRORS_SCHEMA,
   Pipe,
-  PipeTransform,
-  Component,
-  Input
+  PipeTransform
 } from '@angular/core';
 
 class MockNgbActiveModal {
   dismiss() {}
+
   close() {}
 }
 
 class MockCartService {
+  getLoaded() {}
+
   updateEntry(_entryNumber, _updatedQuantity) {}
+
   removeEntry(_entry) {}
 }
 
+const mockItems = [
+  {
+    quantity: 1,
+    entryNumber: 1,
+    id: 111,
+    product: {
+      code: 'CODE1111'
+    }
+  }
+];
+
 @Component({
```

Comment:
to get rid of the error (fixed with `NO_ERROR_SCHEMA`, please add `parent` input in this component mock

Reply:
also, please check if all of the inputs are here